### ISSUE TITLE
Detection of molecule of bacterial/fungal origin

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#18098]
To complete: 

GO:0032492 detection of molecule of oomycetes origin -> no annotations
GO:0032491 detection of molecule of fungal origin -> 7 annotations
GO:0032490 detection of molecule of bacterial origin
GO:0042494 detection of bacterial lipoprotein
GO:0032497 detection of lipopolysaccharide
GO:0070392 detection of lipoteichoic acid
GO:0032499 detection of peptidoglycan
GO:0070340 detection of bacterial lipopeptide
GO:0042496 detection of diacyl bacterial lipopeptide
GO:0042495 detection of triacyl bacterial lipopeptide
GO:0032498 detection of muramyl dipeptide

**Comment by pgaudet** [comment]
Annotation  review ticket: https://docs.google.com/spreadsheets/d/13AUkCP5Fztpe8V9qFMPX-aIkD5ViVbbBIE4L-4Z0SfI/edit#gid=0

**Comment by pgaudet** [comment]
Also: 
GO:0016046 detection of fungus
 GO:0009603 detection of symbiotic fungus
 GO:0001879 detection of yeast 

3 EXP annotations

**Comment by addiehl** [comment]
The structure of some response to organism X/detection of organism X terms was set by #2013 some time ago, though there have been subsequent modifications.  While I agree that detection of specific molecules might better be handled by receptor terms, detection of organisms is more general, as there will be several ways (multiple receptor pathways) in which particular kinds of organisms can be detected.  Separate detection and response exist as detection is an initial type of response and is intended to capture that semantic nuance.

**Comment by ValWood** [comment]
I'm still not understanding  the "detection of organism" (i.e how this isn't recognition by a receptor).

Do you have an example?
If you are worried about the loss of the "organism" the response to x will still exist 

so it would be

"x receptor" or "x signalling" involved in "response to x species"

**Comment by addiehl** [comment]
Of course, I don't annotate any more, and thus don't think in terms of GO-CAM types of coupled annotation.  My fear about the necessity of using a more complex annotation is that some annotators will simply stop at half the annotation, where in the past a pre-composed GO term might have provided the full semantics. 

Also, I do think the very names of these terms, "detection of organism X" is more readily obvious to biologists than "x receptor" or "x signaling" involved in "response to x species", and I worry about the queryability of complex GO-CAM annotations for average users, so I'm not so thrilled to see the detection terms disappear, but I'll save my energies for defending other terms of greater interest to me if nobody else cares about these terms.

**Comment by ValWood** [comment]
So far, since the term GO:0009602   detection of symbiont and the children were established in 2007 NOT A SINGLE ANNOTATION has ever been made to the term or any of its children. 

I suspect this largely because it does not fit the way you would think of a receptor signalling pathway or a term you would look for if you were annotating a host gene?

There are 15,889 annotations to GO:0002221  pattern recognition receptor signaling pathway, and 885 to GO:0038187  pattern recognition receptor activity presumably all of these  (and many many others) could be annotated to "detection of symbiont"  or a descendant but currently, none are.

If I were annotating a host receptor, logically I would think about the MF   (usually PAMP or resistance-gene receptor) and then the immune pathway activated.  

In our tools (Canto)  we would prompt for the  "response to" term.  In Noctua too the MF and BP  would be the primary annotations and the "involved in response to" could be linked as an extension.  

So  an annotation would be something like:

"PAMP receptor activity" (the detection)  part_of "pattern recognition receptor signaling pathway"  (the specific immune response pathway) involved_in "response to fungi"

If you defined "detection of fungus" (def: The series of events in which a stimulus from a fungus is received and converted into a molecular signal) formally, I am sure that it would look  the same?  and this indicates part of the problem with keeping the term, it's another way of  making a statement that we have a standard way of saying with existing MF and BP so it would lead to inconsistency (although in this case it didn't because nobody found/used it).

I agree that searching might be an issue but new tools need to handle this more generally.  

(Note that the above only applies If I have correctly understood the meaning of 'detection of organism')

**Comment by addiehl** [comment]
I'd rather obsolete all the detection terms rather than change them to MF.  As 'detection of other organism' only has 169 annotations (35 experimental), it clearly is not a popular term, and thus it and its children should be expunged from the GO.  Most of those annotations can simply be migrated upward to 'response to other organism' or one of its organism specific children without any major loss of meaning.
